### PR TITLE
<DropdownLayout /> - fix uni driver when using infinite scroll

### PR DESCRIPTION
**Pull request by eugenevwix**
Motivation: When using `infiniteScroll` uni driver methods returns wrong options.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6937/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6937

File: packages/wix-style-react/src/DropdownLayout/DropdownLayout.uni.driver.js [hunk 1]
```diff
@@ -39,9 +39,9 @@ export const dropdownLayoutDriverFactory = base => {
     await base.$(
       `[data-hook=${await getOptionsContainerDataHook()}] [data-hook="${dataHook}"]`,
     );
-  const options = () =>
+  const options = async () =>
     base
-      .$$(`[data-hook=${DATA_HOOKS.DROPDOWN_LAYOUT_OPTIONS}] > *`)
+      .$$(`[data-hook=${await getOptionsContainerDataHook()}] > *`)
```
Comment (mykas) [inline packages/wix-style-react/src/DropdownLayout/DropdownLayout.uni.driver.js:44]: Lets add a unit test that failed before and now is green

**Review by mykas — CHANGES_REQUESTED** [review]

File: packages/wix-style-react/src/DropdownLayout/DropdownLayout.uni.driver.js (hunk 1 above)
Comment (eugenevwix) [reply packages/wix-style-react/src/DropdownLayout/DropdownLayout.uni.driver.js:44]: added

**Review by mykas — APPROVED** [review]
Nice! Thank you

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6937/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6937

**Pull request merged**